examples/experimental/omniglot_protonet_pytorch/README.md

```diff
@@ -0,0 +1,24 @@
+## Prototypical Networks on Omniglot
+This example implements a meta-learning method called prototypical networks (ProtoNets) for few-shot learning on the Omniglot dataset.  ProtoNets often outperform MAML on few-shot learning problems and is one of the most popular methods for meta-learning.
+Please consult the original [paper](http://papers.nips.cc/paper/6996-prototypical-networks-for-few-shot-learning.pdf) and [code](https://github.com/jakesnell/prototypical-networks) published by the author for more details.
+
+### Folders and Files
+* **model_def.py**: Contains the core code for the model and protonet loss function following Determined's PyTorchTrial interface.
+* **data.py**: Contains the code used to generate few-shot learning tasks.
+* **const.yaml**: Contains the configuration for the experiment. 
+* **startup-hook.sh**: Script run before starting the trial that downloads the Omniglot data.
+* **fetch_data.sh**: Script to download the Omniglot data.
+
+### Data
+We use the Omniglot download script from the [meta-blocks](https://github.com/alshedivat/meta-blocks/blob/master/benchmarks/omniglot/fetch_data.sh) repo.  This script is called in `startup-hook.sh` so the data folder is available prior to starting the trial.
```

Comment:
how long does it take to download the data?

Reply:
data download is really fast, less than 15 seconds.